src/Elastic.OpenTelemetry.AutoInstrumentationPlugin/ElasticAutoInstrumentationPlugin.cs

```diff
@@ -0,0 +1,71 @@
+// Licensed to Elasticsearch B.V under one or more agreements.
+// Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information
+
+using System.Diagnostics.Tracing;
+using Elastic.OpenTelemetry.Configuration;
+using Elastic.OpenTelemetry.Extensions;
+using Microsoft.Extensions.Logging;
+using OpenTelemetry.Logs;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+
+namespace Elastic.OpenTelemetry.AutoInstrumentationPlugin;
+
+/// <summary>
+/// Elastic Distribution for OpenTelemetry .NET plugin for Auto Instrumentation.
+/// <para>Ensures all signals are rich enough to report to Elastic</para>
+/// </summary>
+// ReSharper disable once UnusedType.Global
+public class ElasticAutoInstrumentationPlugin
+{
+	private readonly ILogger _logger;
+	private readonly EventListener _eventListener;
+
+	/// <inheritdoc cref="ElasticAutoInstrumentationPlugin"/>
+	public ElasticAutoInstrumentationPlugin()
+	{
+		var options = new ElasticOpenTelemetryBuilderOptions();
+		var (eventListener, logger) = ElasticOpenTelemetryBuilder.Bootstrap(options);
```

Comment:
Any reason not to call the `ElasticOpenTelemetryBuilder` ctor and skip there shared bootstrap?

Reply:
Mainly because `CompositeLogger` and `LogEventListener` are internal and need to be exposed as their types vs here we only need them to be `ILogger` and `EventListener`.

Reply:
The other is that the `ElasticOpenTelemetryBuilder` constructor sets up tracing using `IOpenTelemetryBuilder` using the `IServiceCollection` approach where as the callbacks using from the auto instrumentation still use the global `Sdk.*` setup mechanism. 

The `Bootstrap` here ensures both manual/auto instrumentation share the same logging infrastructure and preamble logging.